backend/src/main/java/gov/cdc/usds/simplereport/service/email/EmailService.java

```diff
@@ -118,30 +119,45 @@ public String send(
         List.of(toEmail), subject, getContentFromTemplate(templateName), attachments);
   }
 
-  public String sendWithDynamicTemplate(
+  public List<String> sendWithDynamicTemplate(
       final String toEmail, final EmailProviderTemplate providerTemplate) throws IOException {
-    return sendWithDynamicTemplate(toEmail, providerTemplate, null);
+    return sendWithDynamicTemplate(List.of(toEmail), providerTemplate, null);
   }
 
-  public String sendWithDynamicTemplate(
-      final String toEmail,
+  public List<String> sendWithDynamicTemplate(
+      final List<String> toEmails, final EmailProviderTemplate providerTemplate)
+      throws IOException {
+    return sendWithDynamicTemplate(toEmails, providerTemplate, null);
+  }
+
+  public List<String> sendWithDynamicTemplate(
+      final List<String> toEmails,
       final EmailProviderTemplate providerTemplate,
       final Map<String, Object> templateVariables)
       throws IOException {
-    Mail mail = new Mail();
-    mail.setFrom(
-        new Email(sendGridProperties.getFromEmail(), sendGridProperties.getFromDisplayName()));
+    List<String> results = new ArrayList<String>();
 
-    mail.setTemplateId(sendGridProperties.getDynamicTemplateGuid(providerTemplate));
+    for (String toEmail : toEmails) {
+      Mail mail = new Mail();
+      mail.setFrom(
+          new Email(sendGridProperties.getFromEmail(), sendGridProperties.getFromDisplayName()));
 
-    Personalization personalization = new Personalization();
-    personalization.addTo(new Email(toEmail));
+      mail.setTemplateId(sendGridProperties.getDynamicTemplateGuid(providerTemplate));
 
-    if (templateVariables != null) {
-      templateVariables.forEach(personalization::addDynamicTemplateData);
+      Personalization personalization = new Personalization();
+      personalization.addTo(new Email(toEmail));
+
+      if (templateVariables != null) {
+        templateVariables.forEach(personalization::addDynamicTemplateData);
+      }
+
+      mail.addPersonalization(personalization);
+
+      String result = emailProvider.send(mail);
+
+      results.add(result == null ? "" : result);
```

Comment:
I'm not sure exactly what the string returned by `emailProvider.send` is supposed to be, and in any case we never use the return value of this `sendWithDyamicTemplate`.